gen/ad_group_mu_ucn

```diff
@@ -56,10 +62,13 @@ foreach my $resourceId ($data->getResourceIds()) {
 
 		my $login = $data->getUserFacilityAttributeValue( member => $memberId, attrName => $A_LOGIN );
 		my $isBanned = $data->getMemberResourceAttributeValue( member => $memberId, resource => $resourceId, attrName => $A_MR_V_IS_BANNED );
+		my $memberStatus = $data->getMemberAttributeValue( attrName => $A_MEMBER_STATUS, member => $memberId );
```

Comment:
I think the code looks okay overall, just a nitpick about the argument order.

Reply:
Thanks, fixed the ordering.